.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 0
+
+  - package-ecosystem: "terraform"
```

Comment:
Do we need terraform when there's none in the repo?

Reply:
Probably not, was mostly using the same config for all BCDA repos, I'll go back through and remove anything that doesn't quite fit